packages/datawell/src/react/__specs__/index.spec.js

```diff
@@ -1,11 +0,0 @@
-import React from 'react'
-import { shallow } from 'enzyme'
-
-import DataWell from '../index.js'
-
-describe('DataWell', () => {
```

Comment:
Why the removal?  "Renders" tests have always been a little hokey, but they are at least some kind of smoke test.

Reply:
idk...seemed pretty basic and there are storyshots that do a better job. I do usually write a test like this(it's in the generator) but it's not consistent for all components. we could decided not to do a "render" test or we could make sure all component have a render test. thoughts?